src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
+    white,
+    black,
+    blue,
+    yellow,
+    orange,
+    red,
+    green,
+    brown;
```

Comment:
```suggestion
    WHITE,
    BLACK,
    BLUE,
    ...
```

элементы списка принято писать капсом